src/test/scripts/get_product_lts_version.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import urllib.request
+import json
+
+known_lts_version = {
```

Comment:
These appear to be the the latest product releases, and not the current LTS versions for each product. Current LTS versions are: `8.13.8`/`7.4.9`/`7.6.7`

Reply:
This part is not needed at all but as currently LTS version of Confluence and Bitbucket does not support Kubernetes, so I added a logic to feed bamboo with a valid release until next LTS release for those products.